src/helpers.js

```diff
@@ -88,11 +88,20 @@ Marionette.normalizeUIKeys = function(hash, ui) {
 // allows for the use of the @ui. syntax within
 // a given value for regions
 // swaps the @ui with the associated selector
-Marionette.normalizeUIValues = function(hash, ui) {
+Marionette.normalizeUIValues = function(hash, ui, properties) {
   _.each(hash, function(val, key) {
     if (_.isString(val)) {
       hash[key] = Marionette.normalizeUIString(val, ui);
     }
+    else if (_.isObject(val) && _.isArray(properties)) {
+      /* Value is an object, and we got an array of embedded property names to normalize. */
+      _.each(properties, function(property) {
+        var propertyVal = val[property];
```

Comment:
I think we can be really sneaky here and use recursion 

```js
val = Marionette.normalizeUIValues(_.pick(val, properties), ui)
```

thus cleaning this up just a touch :dancer: